### ISSUE TITLE
download_historic_data() errors out

### ISSUE THREAD

**Issue by joethorley** [bcgov/rems#7]
```
> library(rems)
> download_historic_data()
Error in `row.names<-.data.frame`(`*tmp*`, value = value) : 
  duplicate 'row.names' are not allowed
In addition: Warning message:
non-unique value when setting 'row.names': ‘3yr_current’ 
> 
```

**Comment by ateucher** [comment]
Thanks for this, I'll look into it.

**Comment by ateucher** [comment]
Should be fixed in a23c72596cee13e0bfdffc0b9166d8da449734c6. Let me know if it works for you

**Comment by joethorley** [comment]
That works - thanks :)
